aip/general/0136.md

```diff
@@ -174,8 +173,21 @@ RPC name itself causes confusion, and can even cause issues for client libraries
 which generate both synchronous and asynchronous methods to call the RPC in some
 languages.
 
+### Choice of HTTP methods for stateless methods
+
+HTTP clients such as browsers often cache `GET` responses as `GET` implies that
+it is doing data retrieval. Stateless methods that are cacheable and send no
+payload **should** use `GET` HTTP method due to its cache-friendly nature. 
+This can reduce the costs for both the server and the client.
+
+If the stateless method is not cacheable by design, it **should** use `POST`
```

Comment:
this isn't necessarily true - you might want to still have a non-cacheable request be `GET`: the method has more to do with the user intent, not just cacheability.

If you can remove this sentence I can LGTM.

Reply:
Done. Thanks!